src/problem_data.jl

```diff
@@ -105,15 +105,14 @@ function load_problem!(pb::ProblemData{T},
     A::SparseMatrixCSC,
     lcon::Vector{T}, ucon::Vector{T},
     lvar::Vector{T}, uvar::Vector{T},
-    con_names::Vector{String}, var_names::Vector{String}
+    var_types::Union{Nothing,Vector{VariableType}}=nothing,
```

Comment:
We should check the length of `var_types` as well

Reply:
Done.